package-lock.json

```diff
@@ -8775,7 +8775,7 @@
 				"eslint-config-prettier": "^8.3.0",
 				"eslint-plugin-import": "^2.25.2",
 				"eslint-plugin-jest": "^27.2.1",
-				"eslint-plugin-jsdoc": "^37.0.3",
```

Comment:
We need to bump this library accordingly with the changes introduced in Gutenberg: https://github.com/WordPress/gutenberg/pull/47490.

Reply:
I created https://github.com/wordpress-mobile/gutenberg-mobile/pull/5437 to bump this library in GB-mobile. @geriux let me know if you could take a quick look. Thanks!